### ISSUE TITLE
Minutes are not rounded properly

### ISSUE THREAD

**Issue by manufont** [en-japan-air/react-intl-formatted-duration#14]
First, let me thank you for providing such a useful component.
I stumbled upon a case where the minutes value is not rounded properly: It renders '1 hour 60 minutes' when it should be '2 hours 0 minutes'. Is that the expected behavior ?

![image](https://user-images.githubusercontent.com/20096663/46960381-fe1eb380-d0d0-11e8-9b7d-77262890af9e.png)

```
<FormattedDuration
  seconds={7179}
  textComponent="span"
  format="{hours} {minutes}"
/>
```

**Comment by piuccio** [comment]
oh interesting. Surely it's not expected, I'll try to have a look these days

**Comment by manufont** [comment]
For anyone wondering, I managed to make it work by using the format `{hours} {minutes} {seconds}` with a value rounded to the closest minute.

```
const stripSeconds = value => Math.round(value / 60) * 60;

[...]

<FormattedDuration
  seconds={stripSeconds(7179)}
  textComponent="span"
  format="{hours} {minutes} {seconds}"
/>
```

**Comment by piuccio** [comment]
Hey, I'm now working on this and I'm wondering what should be the expected result, `7179` is `1h 59m 39s`. Should the library approximate automatically? I'm tempted to go for `1h 59m` with a configurable option to display `2h`

**Comment by manufont** [comment]
Thank you for taking care of this issue.

According to the documentation:

> Seconds is also optional and if missing, minutes will be rounded to the closed value
> ```
> <FormattedDuration seconds={10} textComponent={Text} format="{minutes}" />
> // will render `0 minutes`
> 
> <FormattedDuration seconds={30} textComponent={Text} format="{minutes}" />
> // will render `1 minute`
> 
> <FormattedDuration seconds={70} textComponent={Text} format="{minutes}" />
> // will render `1 minute`
> ```

So I guess the default option should be to round it to `2h`, for consistency with other formats

**Comment by piuccio** [comment]
I've published a fix for this in version `3.0.0` (#15)
